src/pages/pricing.js

```diff
@@ -36,7 +36,7 @@ const PricingBlock = styled(BasePricingBlock)`
 
 const Pricing = () => {
   const { meta, hero, faq, table, pricingAssistant, pricing } =
-    getPricingPageData()
+    pricingPageData()
```

Comment:
You are still calling a function (`()`) but the function has been changed into an object.
Please make sure to test and run the project before submitting a PR 🙏
You can change this one and the other ones in the code 😉